### ISSUE TITLE
Logo on search page "framed" on mobile/iPhone

### ISSUE THREAD

**Issue by meols** [equinor/energyvision#935]

**Comment by meols** [comment]
On iPhone the logo is "framed" when opening the search page 
(tested on iPhone 8+)
![image.png](https://images.zenhubusercontent.com/5fbbf4f2bab8ae744cb6560f/c7902f34-ed3d-4acb-a723-3d7f0bc06857)

**Comment by wenche** [comment]
@meols  It's just the default focus indicator. (Old) iPhones doesn't work in the exact same way as every other device :( 

I couldn't replicate this using Browserstack, but I added a quick fix. If this doesn't solve the problem for you, my recommendation  will be to just let it go as this is really a small issue.

(And the default one, but we are doing some magic to avoid to show the default focus indicators with push and mouse input devices, and instead only show them if you use a keyboard)

**Comment by wenche** [comment]
@meols Soon (approx 10 minutes) to be available for testing in the [test environment ](https://web-global-test-equinor-web-sites-test.c2.radix.equinor.com/)

**Comment by meols** [comment]
Looks good - thanks!
